client/app/projects/profile/profile.controller.js

```diff
@@ -182,7 +182,7 @@ angular.module('observatory3App')
     };
 
     $scope.checkUserProject = function() {
-        $scope.userOnProject = $scope.user.projects.indexOf($scope.project._id) !== -1;
+        $scope.userOnProject = $scope.user.projects.indexOf($scope.project._id) !== -1 || Auth.isMentor($scope.user);
```

Comment:
This doesn't work  - it shows the mentor as being on the project (shows the "leave project" buttons). It would be a separate check on the front end.